packages/react-components/src/components/RadioButtons/RadioButtons.mdx

```diff
@@ -92,4 +92,4 @@ export class RenderedComponent extends React.Component {
   }
 }
 
-<RenderedComponent />
+<RenderedComponent />
```

Comment:
I know this isn't actually changing anything, but is there any way to remove this change?

Reply:
It seemed to want a line break at the end so this should be resolved now.